common/m4/lib/clean-temp.h

```diff
@@ -107,7 +107,9 @@ extern int cleanup_temp_dir_contents (struct temp_dir *dir);
    Return 0 upon success, or -1 if there was some problem.  */
 extern int cleanup_temp_dir (struct temp_dir *dir);
 
+#ifndef __MINGW32__
```

Comment:
This should be fixed when #104 gets merged

Reply:
Upstream (like in [this mirror](https://gitlab.com/libidn/gnulib-mirror/-/blob/master/lib/clean-temp.h)) only includes sys/types.h and expects mode_t to be defined. Is that wrong?